.github/workflows/tests.yml

```diff
@@ -49,6 +51,9 @@ jobs:
   health-score:
     needs: lint-test
     runs-on: macos-latest
+    permissions:
+      checks: write
+      contents: read
```

Comment:
📣 AFAICT "contents" is not required for public repos but seems to be a good practice for making this permission clear.

📚 The "checks" permission is required for the health score: https://github.com/slackapi/slack-health-score